onnxruntime/python/tools/transformers/__init__.py

```diff
@@ -14,3 +14,5 @@
 import gpt2_helper  # noqa: E402, F401
 
 sys.path.append(os.path.join(os.path.dirname(__file__), "models", "t5"))
+
+sys.path.append(os.path.join(os.path.dirname(__file__), "models", "whisper"))
```

Comment:
Not sure if I missed, if there is any doc to show how to use it in command line?

Reply:
Added a document